src/microbe_stage/MicrobeStage.cs

```diff
@@ -192,9 +192,11 @@ public override void _Process(float delta)
 
         if (Player != null)
         {
-            var playerTransform = Player.GlobalTransform;
-            spawner.Process(delta, playerTransform.origin);
-            Clouds.ReportPlayerPosition(playerTransform.origin);
+            DebugOverlays.Instance.ReportPositionCoords(Player.GlobalTranslation);
+            DebugOverlays.Instance.ReportLookingAtCoords(Camera.CursorWorldPos);
```

Comment:
I'd prefer these to have the full "Coordinates" name in it, function names are not space constrained like the GUI where I think having "coords" in the debug view is fine.